api/build-tools/schema-sync.lua

```diff
@@ -0,0 +1,147 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+package.loaded["apisix.core.schema"] = {}
```

Comment:
we need some comment here for why we set `package.loaded`.

and we can use a `for .. do` for this process which is simpler.

eg:

```lua
local fake_module_list = {
    'cjson', 
    'cjson',
    -- ...
}
for _, name in ipairs(fake_module_list) do
    package.loaded[name] = {}
end
```

Reply:
done.